.github/workflows/publish-vropkg.yml

```diff
@@ -0,0 +1,20 @@
+name: publish vropkg
```

Comment:
What do you think about using a single `publish` workflow for everything - vrdt, vropkg, polyglotpkg, etc. You can define each one as separate job.

Reply:
I could. I need some time to investigate can I do it with lerna.

Reply:
I have update the lerna json. We must discuss how would we publish the package.